### PR TITLE
adding panel meeting type TM-3929

### PR DESCRIPTION
**Pull request by SidelineCory24**
Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2439)
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/747)

To-do:
- [x] panel date type data shows in service

### CODE REVIEW

File: migrations/082_panel_meeting_items_update.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+exports.up = function (knex) {
```
Comment (SidelineCory24) [inline migrations/082_panel_meeting_items_update.js:1]: I was not sure if it was best to create a new migration or edit and old one. Based on the naming it looks like we create a new migration, so I went that route.

File: migrations/082_panel_meeting_items_update.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  return knex.schema
+    .alterTable('panelmeetingitems', function (table) {
+      table.string('pmtcode')
```
Comment (SidelineCory24) [inline migrations/082_panel_meeting_items_update.js:4]: verified column shows in table+

File: seeds/030_panelmeetingitems_seed.js [hunk 3]
```diff
@@ -43,6 +43,7 @@ exports.seed = function(knex) {
                         pmicreatedate: createDate,
                         pmiupdateid: _.sample(aoCdoPerdets),
                         pmiupdatedate: updateDate,
+                        pmtcode: _.sample(['ML', 'ID']),
```
Comment (SidelineCory24) [inline seeds/030_panelmeetingitems_seed.js:46]: verified this data is seeded in `table+`

File: src/services/agendas.js [hunk 4]
```diff
@@ -256,6 +256,7 @@ const getAgendaItems = async (ai_id, perdet, pmseqnum) => {
           pmdmdtcode: pmdt.mdtcode,
           pmddttm: pmdt.pmddttm,
           micdesctext: _.find(pmicData, ['miccode', pmi.miccode])['micdesctext'],
+          pmtcode: pmi.pmtcode,
```
Comment (SidelineCory24) [inline src/services/agendas.js:259]: verified this data shows in the console on the `mock` and `be`

**Comment by elizabeth-jimenez** [comment]
You didn't have to add a new col, we already have pmpmtcode in PanelMeetings table

File: seeds/030_panelmeetingitems_seed.js (hunk 3 above)
Comment (SidelineCory24) [reply seeds/030_panelmeetingitems_seed.js]: **new mock**
image

**Comment by SidelineCory24** [comment]
> You didn't have to add a new col, we already have pmpmtcode in PanelMeetings table



> You didn't have to add a new col, we already have pmpmtcode in PanelMeetings table

getting `pmpmtcode` that's already in `pm`

**Comment by elizabeth-jimenez** [comment]
Code looks good and data coming through - waiting to verify in dev1

**Review by elizabeth-jimenez — APPROVED** [review]

**Pull request merged**